### DISCUSSION TITLE
Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<DataTable> & Readonly<DataTableProps>'

### DISCUSSION THREAD

**Discussion by Ahulkv** [primefaces/community#488] [PrimeReact]
Property 'children' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<DataTable> & Readonly<DataTableProps>'
 Getting the above typescript error for Datatable component 
 typescript version : 4.2.4
  "primereact": "^4.2.0",

**Comment by melloware** [comment]
Ancient PrimeReact version does not support modern React if you are trying to upgrade your React Version to say 18 you need to upgrade PrimeReact as well.
